Marlin/Configuration.h

```diff
@@ -1265,7 +1270,7 @@
 // RepRapDiscount FULL GRAPHIC Smart Controller
 // http://reprap.org/wiki/RepRapDiscount_Full_Graphic_Smart_Controller
 //
-//#define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+#define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
```

Comment:
Even if I ignore the changes, git won't.

You may revert these changes in your branch, clean up excess commits with `git rebase -i HEAD~5`, and use `git push -f` to update the PR, if you wish.